x-pack/legacy/plugins/siem/public/containers/detection_engine/signals/api.ts

```diff
@@ -69,3 +81,87 @@ export const updateSignalStatus = async ({
   await throwIfNotOk(response);
   return response.json();
 };
+
+/**
+ * Fetch Signal Index
+ *
+ * @param kbnVersion current Kibana Version to use for headers
+ */
+export const getSignalIndex = async ({
+  kbnVersion,
+  signal,
```

Comment:
nit: param signal is not described in JSDoc (for this, `getUserPrivilege`, and `createSignalIndex` below)

``` ts
 * @param signal AbortSignal for cancelling request
```